content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,250 @@
+   application displays all contracts in a Kanban view, organized by their stage, regardless of
+   status. All contracts can be viewed by changing the filters.
+
+Create a new contract
+=====================
+
+In order for an employee to be paid, an active contract is required. If a new contract is needed,
+click the :guilabel:`Create` button on the contracts dashboard.
+
+.. image:: contracts/new-contract-button.png
+   :align: center
+   :alt: Click the NEW smart button to create a new contract.
+
+A contract form appears where the information can be entered. Required fields are underlined in
+bold.
+
+Required fields
+---------------
+
+.. image:: contracts/required-fields.png
+   :align: center
+   :alt: New contract form to be filled in when creating a new contract.
+
+- :guilabel:`Contact Reference`: Type in the name or title for the contract, such as `John Smith
+  Contract`.
+- :guilabel:`Company`: Select the company the contract applies to by clicking on the drop-down
+  menu. A new company can be created by typing the name in the box, then clicking either
+  :guilabel:`Create` to create the new company, or :guilabel:`Create and Edit` to create the new
+  company and edit the company details.
+- :guilabel:`Salary Structure Type`: Select one of the salary structure types from the drop-down
+  menu. The default salary structure types are :guilabel:`Employee` or :guilabel:`Worker`. A new
+  salary structure type can be created by typing the name in the field.
+- :guilabel:`Start Date`: The date the contract starts. Choose a date by clicking on the drop-down
+  menu, navigating to the correct month and year by using the arrow icons, then clicking on the
+  date.
+- :guilabel:`Working Schedule`: Select one of the working schedules from the drop-down menu.
+
+.. tip::
+   The :guilabel:`Working Schedule` drop-down menu displays all the working times for the selected
+   :guilabel:`Company`.
+   To modify or add to this list, go to :menuselection:`Payroll --> Configuration --> Working Times`
+   and either :guilabel:`Create` a new working time or click on an existing working time and
+   edit it by clicking :guilabel:`Edit`.
+
+Optional fields
+---------------
+
+.. image:: contracts/optional-fields.png
+   :align: center
+   :alt: Optional fields for a new contract.
+
+- :guilabel:`Employee`: Name of the employee that the contract applies to.
+- :guilabel:`Department`: The department the contract applies to.
+- :guilabel:`Job Position`: The specific job position the contract applies to.
+- :guilabel:`Contract Type`: Choose from :guilabel:`CDI`, :guilabel:`CDD`, or :guilabel:`PFI` from
+  the drop-down menu.
+
+   - :guilabel:`CDI` is an open-ended contract with only a start date but no end date.
+   - :guilabel:`CDD` is a contract with both a start date and an end date.
+   - :guilabel:`PFI` is a Belgian-specific contract used when hiring employees that need training,
+     and covers the training period specifically.
```

Comment:
Bullet points indented by an extra space

Reply:
I made this indented one more since these 3 items are explaining the options on the previous bullet.  So since it's linked/not part of the same list, but a sub-list of the list, I indented it.  If that doesn't make sense, I can remove, but I thought the format made sense,